src/psyclone/tests/psyir/nodes/acc_directives_test.py

```diff
@@ -61,7 +61,7 @@
 def setup():
     '''Make sure that all tests here use a new Config instance.'''
     Config._instance = None
-    yield()
+    yield
```

Comment:
Out of curiosity - what is the difference? We have 22 `yield()`s and 7 `yield`s. Nothing wrong, I am just curious :)

Reply:
Since I updated to python3.10 and pycodestyle 2.9 I was getting:
`src/psyclone/tests/psyir/nodes/acc_directives_test.py:64:10: E275 missing whitespace after keyword`

yield is like a return so I think `yield()` looked like a function but actually is returning a empty tuple without the whitespace between the keyword and the tuple, which I don't think is what we meant.

I updated all of them now.

Reply:
Thanks! I used this as a reason to update my pylint, and get the same now, thanks (I didn't even wanted you to fix them all ... but this is certainly appreciated!!).